modules/Emails/EmailsController.php

```diff
@@ -722,7 +722,7 @@ protected function userIsAllowedToSendEmail($requestedUser, $requestedInboundEma
         }
 
         $isAllowedToUseOutboundEmail = false;
-        if ($outboundEmailAccount->type === 'system') {
+        if ($outboundEmailAccount->type === 'system' || $outboundEmailAccount->type === 'system-override') {
```

Comment:
Feature is removed in https://github.com/salesagility/SuiteCRM/pull/3873/ so we don't need it